### DISCUSSION TITLE
Nova community Discord

### DISCUSSION THREAD

**Discussion by apexskier** [apexskier/nova-typescript#76] [General]
If you're interested in joining a new community discord to chat about Nova, extension development, or whatever, reach out to me and I'll send you an invite. (thanks to @henrikdahl for setting up it)

**Comment by jasonplatts** [comment]
Hi Cameron. I would love an invite. Working on a Tailwind CSS extension for Nova.

**Comment by apexskier** [comment]
@jasonplatts send me a quick email and I'll send you an invite link (my email's on my GitHub profile)

**Comment by khrome83** [comment]
I am interested, for sure.

**Comment by olierxleben** [comment]
Interested, does it still exist?

**Comment by jasonplatts** [comment]
Hi @olierxleben. It exists, but I think most of us are using the Nova dev forum. I'm sure if you email @apexskier, he would be happy to provide you an invite link (see message above).
